install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -236,6 +236,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: SDS_ENABLED
```

Comment:
@JimmyCYJ Please update this PR's description to link to the equivalent PR that updates the new installer. If you won't update the new installer, please explain why.

Reply:
@myidpt Could you provide any context about equivalent changes to new installer?